infra/charts/turing/templates/cluster-role-binding.yaml

```diff
@@ -3,6 +3,11 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: {{ template "turing.serviceAccount.name" . }}-role-binding
+  labels:
+{{ include "turing.labels" . | indent 4 }}
+{{- if .Values.turing.labels }}
```

Comment:
This if block can also be moved into the `_helpers.tpl` file, under the definition for "turing.labels", right?